test/srt/lora/test_lora.py

```diff
@@ -90,17 +92,32 @@ def _run_lora_multiple_batch_on_model_cases(self, model_cases: List[LoRAModelCas
                 )
 
                 # Initialize runners
+                spec_args = (
+                    {}
+                    if not use_spec_decoding
+                    else {
+                        "speculative_algorithm": "NGRAM",
+                        "speculative_num_draft_tokens": 5,
+                        "speculative_ngram_min_match_window_size": 2,
+                        "speculative_ngram_max_match_window_size": 15,
+                    }
+                )
                 srt_runner = SRTRunner(
                     base_path,
                     torch_dtype=torch_dtype,
                     model_type="generation",
                     lora_paths=[lora_adapter_paths[0], lora_adapter_paths[1]],
                     max_loras_per_batch=len(lora_adapter_paths) + 1,
                     sleep_on_idle=True,  # Eliminate non-determinism by forcing all requests to be processed in one batch.
-                    attention_backend="torch_native",
+                    enable_deterministic_inference=True,
+                    disable_cuda_graph=True,
```

Comment:
Note: cuda_graph is actually implicitly disabled in main because the usage of "torch_native". For some reasons, current main cannot pass this test when cuda graph is enabled. https://github.com/sgl-project/sglang/issues/12914